components/TitleBar.qml

```diff
@@ -56,6 +56,9 @@ Rectangle {
     property bool showMoneroLogo: false
     property bool small: false
     property alias titleBarGradientImageOpacity: titleBarGradientImage.opacity
+    property bool orange: false
+    property string buttonHoverColor: "#262626"
+    property string buttonHoverColorOrange: "#44FFFFFF"
```

Comment:
```suggestion
    property string buttonHoverColorOrange: "#44FFFF"
```

Reply:
`#44FFFFFF` is actually correct, try comparing the two.

Reply:
http://doc.qt.io/qt-5/qml-color.html